src/Client/Player/Player.py

```diff
@@ -9,14 +9,15 @@
 
 from src.utils.BaseMessage import BaseMessage
 
+from src.utils.general_constants import SCREEN_HEIGHT
+
 
 class PicFile(Enum):
     ORANGE = "img/orange_test_image.png"
     APPLE = "img/apple_test_image.png"
 
 
 class Player(pg.sprite.Sprite):
-    SCREEN_HEIGHT = 600  # TODO use general constant
     GREEN = (0, 255, 0)
     RED = (255, 0, 0)
     WHITE = (255, 255, 255)
```

Comment:
It was not explicitly mentioned in each row, but I would consider useful to use general constants for the colours as well. Won't you?

Reply:
It occured to me as well. So be it.
Oh I just noticed that there is an FPS too. It's the most important for that to be general...